question-5/question-5.php

```diff
@@ -7,11 +7,32 @@
  * 1. Register custom post type
  */
 
-// Your code here
+// Register Custom Post Type
+function lion_interview() {
+  $args = array(
+    'public'                => false,
+    'show_ui'               => true,
+    'show_in_menu'          => true,
```

Comment:
```suggestion
    'show_in_menu'          => true,
    'supports'              => array( 'title', 'editor', 'author' ), 
```
The default value for the `supports` parameter is just `array( 'title', 'editor' )`, so you support `author` as well, you need to manually specify it.

Reply:
@bryanatlion ah good catch. Fixed. 

Alternatively I would’ve entirely restricted the CPT and provided access on a wp_role level (outside scope of question reqs).